assets/js/sidebar/sidebar-version-select.js

```diff
@@ -70,3 +70,29 @@ function handleVersionSelected (event) {
       }
     })
 }
+
+/**
+ * Opens the version select if available.
+ * Only focuses the version select if
+ *   - the browser's HTMLSelectElement lacks the showPicker method
+ *   - there has been transient user interaction
+ */
+export function openVersionSelect () {
+  const select = qs(VERSIONS_DROPDOWN_SELECTOR)
+
+  if (select) {
+    select.focus()
+
+    // Prevent subsequent 'v' press from submitting form
```

Comment:
This confused me, but the `v` submits the form. You can test this on the [main hexdocs page for Elixir](https://hexdocs.pm/elixir/Kernel.html).

Click the version select without selecting a version, press `escape` (the select is still focused), press `v`, and it will take you to the version below your current selection.